src/config.cpp

```diff
@@ -252,6 +252,7 @@ const std::map<std::string, AnyParser> CONFIG_KEY_MAP = {
   { "missing_probes", CONFIG_FIELD_PARSER(missing_probes) },
   { "print_maps_on_exit", CONFIG_FIELD_PARSER(print_maps_on_exit) },
   { "use_blazesym", CONFIG_FIELD_PARSER(use_blazesym) },
+  { "show_debug_info", CONFIG_FIELD_PARSER(show_debug_info) },
```

Comment:
missing adoc change?

Reply:
It seems like we're not advertising blazesym in the adoc yet. Maybe we're waiting till it becomes the default?